src/main/java/core/basesyntax/strategy/impl/BalanceTransactionHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.strategy.TransactionHandler;
+
+public class BalanceTransactionHandler implements TransactionHandler {
+    @Override
+    public int getAmount(int amount) {
+        return amount;
```

Comment:
handlers should call dao methods and pass correct amount
we talked about this on QnA